recipe/meta.yaml

```diff
@@ -19,45 +19,54 @@ outputs:
     script: build_base.bat  # [win]
     script: build_base.sh  # [not win]
     build:
-      skip: true  # [py<37]
+      skip: true  # [py<38]
       ignore_run_exports:
         - tk  # [linux]
+        - zlib
     requirements:
       build:
+        - python
+        - cross-python_{{ target_platform }}       # [build_platform != target_platform]
+        - numpy
         - pkg-config  # [not win]
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
       host:
         - python
-        - pip
+        - pip >=9.0.1
         - certifi >=2020.06.20
-        - freetype >=2.3
+        # freetype.pc requires zlib.pc
+        - freetype
+        - zlib
         # an explanation to the utilization of numpy can be found
         # here: https://github.com/AnacondaRecipes/matplotlib-feedstock/pull/13#issuecomment-1296704563
         - numpy   1.19  # [not (osx and arm64) and (py==37 or py==38 or py==39)]
         - numpy   1.21  # [(osx and arm64) and (py==38 or py==39)]
         - numpy   1.21  # [py==310]
         - setuptools
-        - setuptools_scm >=4
+        - setuptools_scm >=7
         - setuptools_scm_git_archive
         - tk  # [linux]
+        - wheel
       run:
         - python
+        - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx and x86_64]
         - cycler >=0.10
+        - contourpy >=1.0.1
         - fonttools >=4.22.0
         - freetype >=2.3
         - kiwisolver >=1.0.1
+        - numpy >=1.19
         - {{ pin_compatible('numpy') }}
         - packaging >=20.0
         - pillow >=6.2.0
-        - pyparsing >=2.2.1
+        - pyparsing >=2.3.1
```

Comment:
`pyparsing` is still on version 2.2.1: https://github.com/matplotlib/matplotlib/blob/v3.6.2/setup.py#L316

Reply:
Changed the pinnings for `pyparsing` back to 2.2.1. Thanks!